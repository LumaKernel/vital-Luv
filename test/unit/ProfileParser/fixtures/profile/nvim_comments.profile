SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_comments.vim
Sourced 1 time
Total time:   0.000095
 Self time:   0.000095

count  total (s)   self (s)
                            
                            " comment1
                              " multibyte : コメント
                              "
                              " no spaces
                              " spaces
                              
                              
                              
                            
                                      "      nasty    comment
                                      "
                            

