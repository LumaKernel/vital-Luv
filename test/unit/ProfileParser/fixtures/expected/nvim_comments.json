[{"path":"/test/test_comments.vim","total_time":7.0e-5,"self_time":7.0e-5,"type":"script","lines":[{"content":""},{"content":"\" comment1"},{"content":"  \" multibyte : コメント"},{"content":"  \""},{"content":"  \" no spaces"},{"content":"  \" spaces"},{"content":"  "},{"content":"  "},{"content":"  "},{"content":""},{"content":"          \"      nasty    comment"},{"content":"          \""},{"content":""}],"count":1}]
